terraform/variables.tf

```diff
@@ -24,8 +24,8 @@ variable "postgres_config" {
     instance_class              = string
     identifier                  = string
     db_name                     = string
-    username                    = string
-    password                    = string
+    username                    = optional(string)
+    password                    = optional(string)
     port                        = optional(number)
```

Comment:
What happens if these aren't provided? I assume the setup will fail, in which case they probably shouldn't be optional?